packages/authentication/src/metadata.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as assert from 'assert';
+import 'reflect-metadata';
```

Comment:
Please add `reflect-metadata` to `packages/authentication/package.json` as a `dependency`. See e.g. [here](https://github.com/strongloop/loopback-next/blob/a422dfb699f512f2daf2692b5cf56395c2425c5b/packages/context/package.json#L15)